sdk/identity/identity/src/credentials/managedIdentityCredential/appServiceMsi2017.ts

```diff
@@ -43,6 +44,9 @@ function prepareRequestOptions(
   if (clientId) {
     queryParameters.clientid = clientId;
```

Comment:
Docs for IMDS suggest this should be `client_id` and not `clientid`. Have we verified this option is working?

Reply:
`client_id` would be necessary for App Service vs 2019, but in JS we only have support for App Service 2017, which uses `clientid`.

Reply:
The reason I mentioned it is because the same docs you linked for `mi_res_id` (https://docs.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal%2Chttp) also say `client_id` and not `clientid`. Are there any docs that show the query parameter docs version-by-version? That page only mentions the 2017 version of the service as an "older version" but doesn't link to any specs.

Reply:
I’m not sure where to find the docs at the moment, but we just got approval to move from App Service 2017 to App Service 2019, so (in a separate PR) I will do the change you suggested 🙂